tests/integration/components/slugged-route-model-details-test.js

```diff
@@ -1,17 +1,35 @@
 import { moduleForComponent, test } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
 import { mockRouter } from 'code-corps-ember/tests/helpers/mock-routing';
+import PageObject from 'ember-cli-page-object';
+import sluggedRouteModelDetailComponent from 'code-corps-ember/tests/pages/components/slugged-route-model-details';
+
+let page = PageObject.create(sluggedRouteModelDetailComponent);
```

Comment:
If you're going to use the full name vs simply `component` then make sure it matches the name of the component (details vs detail).